config/templates/cloud-config-controller

```diff
@@ -723,7 +903,7 @@ write_files:
         }
 {{ end }}
 
-  - path: /srv/kubernetes/manifests/kube-dns-autoscaler-de.yaml
+  - path: /srv/kubernetes/manifests/kube-dns-rc.yaml
```

Comment:
Thanks for rebasing but this should be `kube-dns-autoscaler-de.yaml` as kube-dns has been migrated from `rc` to `de` and is at https://github.com/coreos/kube-aws/pull/124/files#diff-a6019b6709ad6c3c74954ac000740bc5R942!